db/localMigration.sql

```diff
@@ -9,6 +9,10 @@
 INSERT INTO "public"."blocks" ("id", "block_id", "parent_id", "hash", "parent_hash", "state_root", "extrinsics_root", "count_extrinsics", "count_extrinsics_unsigned", "count_extrinsics_signed", "count_extrinsics_error", "count_extrinsics_success", "count_events", "datetime", "spec_version_id", "created_at", "updated_at") VALUES
 ('-1', -1, -1, '0x2', '0x6', '0xc', '0xb0', 1, 1, 0, 0, 1, 1, now(), '3000', now(), now()) ON CONFLICT(id) DO NOTHING;
 
+--Add entry for 0x0100
```

Comment:
I think this comment is off.

If you know them, maybe these should be "add entry for Alice", "add entry for Bob" etc. I am not sure about the 0x000 account though.

If you are not sure OK to leave as is

Reply:
I'm not sure which belongs to whom so I left 0x00, 0x01 ...